packages/hint-sri/src/hint.ts

```diff
@@ -120,14 +120,14 @@ export default class SRIHint implements IHint {
      *
      * More info in https://w3c.github.io/webappsec-subresource-integrity/#is-response-eligible
      */
-    private async isEligibleForIntegrityValidation(evt: FetchEnd, urls: URLs): Promise<boolean> {
+    private isEligibleForIntegrityValidation(evt: FetchEnd, urls: URLs): Promise<boolean> {
         debug('Is eligible for integrity validation?');
 
         const { element, resource } = evt;
         const resourceOrigin: string = new URL(resource).origin;
 
         if (urls.origin === resourceOrigin) {
-            return true;
+            return Promise.resolve(true);
```

Comment:
We should remove all the `Promise.resolve` that are no longer necessary. I've opened https://github.com/webhintio/hint/issues/2035 to track this so no need to do anything in this PR.